source/scenario/introScreenScenario.vwf.yaml

```diff
@@ -17,7 +17,7 @@ extends: source/scenario/scenario.vwf
 properties:
   scenarioName: introScreenScenario
 
-  nextScenarioPath: "scenario1a"
+  nextScenarioPath: "scenario1h"
```

Comment:
Ahhh... let's not do this.  I can jump ahead using the option in settings if I need to.